### DISCUSSION TITLE
input type=radio怎么变成这个样子了？

### DISCUSSION THREAD

**Discussion by Nyazira** [layui/layui#1973] [问答]
Edge中：
![image](https://github.com/layui/layui/assets/26781505/85647856-0ec9-44a0-b053-6469b3e38405)
火狐：
![image](https://github.com/layui/layui/assets/26781505/5d0f09ad-455c-45f3-a7e0-eab9d7e54da0)
layui版本2.9.10

**Comment by LSL1618** [comment]
字体文件没加载到？

**Reply by Nyazira** [reply]
好像确实是这个问题
